src/components/header/template.test.js

```diff
@@ -232,11 +239,4 @@ describe('header', () => {
       expect($('.hmrc-banner > .hmrc-organisation-logo > p.govuk-body-s').text().trim()).toEqual('Cyllid a Thollau EM');
     });
   });
-
-  describe('additional banners block', () => {
```

Comment:
@kyle-bowden is this removal an unrelated change?

Reply:
@oscarduignan thanks for spotting this, this was removed accidentally.